src/js/view/controls/controls.js

```diff
@@ -495,6 +511,10 @@ export default class Controls extends Events {
             infoOverlay.destroy();
         }
 
+        if (shortcutsTooltip) {
+            shortcutsTooltip.destroy();
```

Comment:
I noticed that this shortcutsTooltip is also part of the rightClickMenu (rightclick.js) and I was wondering if it is better to destroy it with the rightClickMenu or within controls? Granted controls is where it is created and info-overlay follows the same pattern.

Reply:
i think destroying it where it is created makes the most sense

Reply:
There's a keyboard shortcut for opening the menu, so it isn't actually dependent on right click.